### PR TITLE
CI to include manta repo when pallet-manta-pay is updated

### LINKED ISSUE #54: CI to include `manta` repo
is it possible to add something like
```
# if PR is for calamari branch...
git clone https://github.com/Manta-Network/Manta
cd Manta
# prob need to change the `pallet-manta-pay` link in Cargo.toml to current PR?
cargo build --release
cargo build --release --features runtime-benchmark
```

### PR DESCRIPTION
**Pull request by ghzlatarev**
closes #54 

Add additional CI to checkout and build the Manta repository when pallet-manta-pay is updated:

1. git clone https://github.com/Manta-Network/Manta
2. cd Manta
3. replace line in Cargo.toml file with ```sed```
4. cargo build --release --all-features

### CODE REVIEW

File: .github/workflows/check.yml [hunk 1]
```diff
@@ -46,17 +46,27 @@ jobs:
...
+      #   run: |
+      #     cargo clippy --release --all
+      # - name: Check Build
+      #   run: |
+      #     cargo check --release
+      #     cargo +nightly check --release --features runtime-benchmarks
+      # - name: Run Tests
+      #   run: |
+      #     cargo run --release --bin param_gen
+      #     cargo test --all-features --release
```
Comment (ghzlatarev) [inline .github/workflows/check.yml]: Revert all the commented code.

**Comment by ghzlatarev** [comment]
One problem is this adds ~30 minutes to our check job,

It could made an optional job, that can be triggered manually through UI or maybe with some keyword in the commit message ?

**Comment by zhenfeizhang** [comment]
great work.

> It could made an optional job, that can be triggered manually through UI or maybe with some keyword in the commit message ?

Just throwing out ideas. How about:
- only runs on PRs
- can be disabled with a keyword

So when we work on a PR we can disable it till it is ready to merge?

**Review by zhenfeizhang — APPROVED** [review]
LGTM

**Comment by ghzlatarev** [comment]
> great work.
> 
> > It could made an optional job, that can be triggered manually through UI or maybe with some keyword in the commit message ?
> 
> Just throwing out ideas. How about:
> 
> * only runs on PRs
> * can be disabled with a keyword
> 
> So when we work on a PR we can disable it till it is ready to merge?

* It's only running on PRs. On push should happen only when a PR is merged.
* The keywords thing needs to be researched. Will log it for now. If the waiting becomes too much, I will revisit it.

**Pull request merged**